File: backEndWarehouse/src/main/java/com/managementsystem/service/ReservoirareaService.java
package com.managementsystem.service;

import com.managementsystem.entity.Reservoirarea;
import com.baomidou.mybatisplus.extension.service.IService;

import java.util.List;


public interface ReservoirareaService extends IService<Reservoirarea> {

    List<Reservoirarea> list();

    int addReservoirarea(Reservoirarea reservoirarea);

    int editReservoirarea(Reservoirarea reservoirarea);

    int deleteReservoirarea(String raId);

    List<Reservoirarea> getReservoirareaByWhId(String whId);

}

File: warehouse/src/service/reservoirareaService.js
//管理承运商数据
import axios from 'axios'

export async function getAllReservoirarea() {
    let result = await axios.get('http://localhost:8181/backEndWarehouse/reservoirarea/findAllReservoirarea')
    return result.data.data;
}

export async function addReservoirarea(reservoirarea){
    let result = await axios.post('http://localhost:8181/backEndWarehouse/reservoirarea/addReservoirarea', reservoirarea);
    return result.data;
}

export async function editReservoirarea(reservoirarea){
    let result = await axios.post('http://localhost:8181/backEndWarehouse/reservoirarea/editReservoirarea',reservoirarea)
    return result.data;
}

export async function deleteReservoirarea(raId) {
    let result = await axios.delete('http://localhost:8181/backEndWarehouse/reservoirarea/deleteReservoirarea/'+raId);
    return result.data;
}


export async function getReservoirareaByWhId(whId) {
    let result = await axios.get('http://localhost:8181/backEndWarehouse/reservoirarea/getReservoirareaByWhId',{
        params:{
            whId,
        }
    })
    return result.data;
}
